tests/integration/nm/testlib.py

```diff
@@ -25,3 +26,11 @@ def mainloop():
     yield
     success = mloop.run(timeout=5)
     assert success, 'Mainloop error: ' + mloop.error
+
+
+def mainloop_run(func):
```

Comment:
On first thought, it should be `run_mainloop`

Reply:
I think decorators are not supposed to be verbs.
They are usually used to add a property/behaviour to a function/method/class.

Reply:
I see.